src/app/machines/views/MachineList/MachineListControls/MachinesFilterAccordion/MachinesFilterOptions/MachinesFilterOptions.tsx

```diff
@@ -33,7 +33,8 @@ const MachinesFilterOptions = ({
   setSearchText,
 }: Props): JSX.Element => {
   const dispatch = useDispatch();
-  const filterOptions = useSelector((state: RootState) =>
+
+  let filterOptions = useSelector((state: RootState) =>
```

Comment:
We should never reassign the value of a redux selector. Why don't you move the additional logic to the selector itself.